static/js/modules/nootils.js

```diff
@@ -6,8 +6,10 @@ module.exports = function(settings) {
   return {
     isTimely: function(date, event) {
       var due = new Date(date);
-      var start = this.addDate(null, event.start);
-      var end = this.addDate(null, (event.end + 1) * -1);
+      var start = new Date();
+      start.setUTCDate(start.getUTCDate() + event.start);
```

Comment:
can you reuse `addDate` here, and below? Or is that not what's going on?
(these date manipulations are pretty obscure to me)